k8sh

```diff
@@ -125,7 +127,7 @@ k8sh_init() {
   reloadExtensions
 
   # Set up PS1 prompt
-  export PS1="($PS_CONTEXT_COLOR\$(ct)$PS_RESTORE/$PS_NAMESPACE_COLOR\$KUBECTL_NAMESPACE$PS_RESTORE) \W ${PS_LPURPLE}\$${PS_RESTORE} "
+  export PS1="($PS_CONTEXT_COLOR\$KUBECTL_CONTEXT$PS_RESTORE/$PS_NAMESPACE_COLOR\$KUBECTL_NAMESPACE$PS_RESTORE) \W ${PS_LPURPLE}\$${PS_RESTORE} "
 
   echo ""
   echo -e "Context: $CONTEXT_COLOR$(ct)$RESTORE"
```

Comment:
Thanks for the PR! This is a great idea. This line, displayed at launch, should just display the current context. Its now displaying the list since ct was changed. Could you swap this out with the $KUBECTL_CONTEXT? Thanks